src/modules/payer-svc/payer-svc.service.test.ts

```diff
@@ -352,4 +353,38 @@ describe('PayerService', () => {
       service.sendSmsVoucher('shortenHash', authUser),
     ).rejects.toThrow(new ForbiddenException(_403.ONLY_OWNER_CAN_SEND_VOUCHER));
   });
+
+  // it("should save information when we scan KYC face", async () => {
+  //   const authUser: JwtClaimsDataDto = {
+  //     sub: 'anotherPayerId',
+  //     type: UserRole.PAYER,
+  //     phoneNumber: mockPayer.user.phoneNumber,
+  //     names: `${mockPayer.firstName} ${mockPayer.lastName}`,
+  //     status: UserStatus.ACTIVE
+  //   };
+
+  //   const kycData: KYCDto = {
+  //     expire: "2017/05",
+  //     cardID: "ABCD",
+  //     birthday: "2002-05-10",
+  //     kyc: true
+  //   }
+  //   expect(async () =>
+  //     service.updateKYC(authUser, kycData),
+  //   ).rejects.toThrow(new BadRequestException(_403.ACCESS_NOT_ALLOWED));
+  // });
+
+  it('should retrieve if payer has already fill his KYC', async () => {
+    const authUser: JwtClaimsDataDto = {
+      sub: 'anotherPayerId',
+      type: UserRole.PAYER,
+      phoneNumber: mockPayer.user.phoneNumber,
+      names: `${mockPayer.firstName} ${mockPayer.lastName}`,
+      status: UserStatus.ACTIVE,
+    };
+
+    expect(async () => service.checkKyc(authUser)).rejects.toThrow(
```

Comment:
you should do the following:
```
const res = await service.checkKyc(authUser);
expect(res).toEqual(mockPayer.kyc);
```

Since the mock function for payerRepository.findOne is resolving to mockPayer